examples/experimental/i3s-17-and-debug/components/debug-panel.js

```diff
@@ -133,6 +135,13 @@ export default class DebugPanel extends PureComponent {
     });
   }
 
+  toggleSemanticValidator() {
```

Comment:
```suggestion
  _onToggleSemanticValidator() {
```

Reply:
I see. We will rename them all in a separate PR